src/main/java/core/basesyntax/validation/DataValidator.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.validation;
+
+import core.basesyntax.model.Operation;
+import java.util.List;
+
+public class DataValidator {
+    public void validate(List<String> inputData) {
```

Comment:
The method validate does not have a return type, it should have a boolean return type to indicate whether the validation was successful or not.